src/CovidSim.cpp

```diff
@@ -705,14 +704,14 @@ void ReadParams(char* ParamFile, char* PreParamFile)
 		GetInputParameter(PreParamFile_dat, AdminFile_dat, "Age distribution of population", "%lf", (void*)P.PropAgeGroup[0], NUM_AGE_GROUPS, 1, 0);
 		t = 0;
 		for (i = 0; i < NUM_AGE_GROUPS; i++)
+		{
 			t += P.PropAgeGroup[0][i];
-		for (i = 0; i < NUM_AGE_GROUPS; i++)
 			P.PropAgeGroup[0][i] /= t;
-		t = 0;
-		for (i = 0; i < NUM_AGE_GROUPS; i++)
+
+			t = 0;
 			if (P.AgeSusceptibility[i] > t) t = P.AgeSusceptibility[i];  //peak susc has to be 1
-		for (i = 0; i < NUM_AGE_GROUPS; i++)
 			P.AgeSusceptibility[i] /= t;
+		}
```

Comment:
I think this changes behaviour.

Loop at lines 706-708, tells us how many people there are in the age group.
Loop 709-710, then scales everything
Loop 712-713 finds the max age susceptibility
Loop 714-715 then scales everything.

Reply:
You are right. I missed that the last `t` from the first loop is used in the next loops. I'll correct those changes.